xbmc/TextureCacheJob.cpp

```diff
@@ -32,7 +32,7 @@
 #include "FileItem.h"
 #include "music/MusicThumbLoader.h"
 #include "music/tags/MusicInfoTag.h"
-#if defined(HAS_OMXPLAYER)
+#if defined(TARGET_RASPBERRY_PI)
```

Comment:
I disagree, they allow selectively disabling features one might not want for one reason or another.
Simply replacing everything with TARGET_RASPBERRY_PI defeats the purpose of this PR and is a regression.